content/management/limits/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+weight: 40
+category: "management"
+title: "Limits"
+tagline: "Resource limits in LiveChat Platform via dedicated APIs"
+desc: "There are some limitations to amount of different resources that can be created, based on license plan."
+---
+
+# Introduction
+
+Based on the <a href="https://www.livechat.com/pricing/" target="_blank">billing plan</a>, the LiveChat Platform limits the number of resources that can be created.
+
+# Resource Limits
+
+| Limited resource              | Starter | Team | Business | Enterprise |
+|-------------------------------|---------|------|----------|------------|
+| Agents                        | 400     | 400  | 400      | 400        |
+| Chat archives history         | 2m      | 1y   | 2y       | unlimited  |
+| Chat queue                    | 10      | 50   | 100      | 100        |
+| Chat queue (continuous chats) | 150     | 150  | 150      | 150        |
+| Groups                        | 1       | 20   | 40       | 80         |
+| Groups assigned to an agent   | 1       | 10   | 20       | 40         |
+| Groups in group chooser       | 1       | 25   | 25       | 25         |
+| Number of banned visitors     | 50      | 200  | 400      | 400        |
+| Tracked visitors              | 400     | 400  | 400      | 400        |
+
+There is a new type of error, `limit_reached`. Our APIs will return it when you try to create a resource over the limit.
+
+Affected methods are:
```

Comment:
```suggestion
```

Reply:
I'd do it this way. "Affected" sounds as if these methods were broken or something :D